.github/workflows/publish.yml

```
@@ -0,0 +1,51 @@
+name: Publish Containers
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Setup Golang
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.16.6'
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: lfedge/edge-home-orchestration-go
```

If someone wants to test this from your Github account, you must change the lfedge to your Docker Hub ID. 
```suggestion
          images: <Docker Hub ID>/edge-home-orchestration-go
```